__tests__/Users.js

```diff
@@ -0,0 +1,237 @@
+import '@testing-library/jest-dom/extend-expect';
+
+import { fireEvent, wait, within } from '@testing-library/react';
+import { forceRemoveAll } from 'lib/flashMessage';
+import nock from 'nock';
+import { StatusCodes } from 'shared';
+import { UsersRoutes } from 'shared/constants/routes';
+import {
+  API_ENDPOINT,
+  appCatalogsResponse,
+  AWSInfoResponse,
+  getMockCall,
+  gsOrgResponse,
+  invitesResponse,
+  ORGANIZATION,
+  orgResponse,
+  orgsWithGSResponse,
+  USER_EMAIL,
+  userResponse,
+  usersResponse,
+  V4_CLUSTER,
+  v4AWSClusterResponse,
+  v4AWSClusterStatusResponse,
+  v4ClustersResponse,
+} from 'testUtils/mockHttpCalls';
+import { renderRouteWithStore } from 'testUtils/renderUtils';
+
+describe('Users', () => {
+  beforeAll(() => {
+    nock.disableNetConnect();
+  });
+
+  afterAll(() => {
+    nock.enableNetConnect();
+  });
+
+  // Responses to requests
+  beforeEach(() => {
+    getMockCall('/v4/info/', AWSInfoResponse);
+    getMockCall('/v4/user/', userResponse);
+    getMockCall('/v4/users/', usersResponse);
+    getMockCall('/v4/organizations/', orgsWithGSResponse);
+    getMockCall('/v4/clusters/', v4ClustersResponse);
+    getMockCall(`/v4/clusters/${V4_CLUSTER.id}/`, v4AWSClusterResponse);
+    getMockCall(
+      `/v4/clusters/${V4_CLUSTER.id}/status/`,
+      v4AWSClusterStatusResponse
+    );
+    getMockCall(`/v4/organizations/${ORGANIZATION}/`, orgResponse);
+    getMockCall(`/v4/organizations/giantswarm/`, gsOrgResponse);
+    getMockCall(`/v4/organizations/${ORGANIZATION}/credentials/`);
+    getMockCall(`/v4/organizations/giantswarm/credentials/`);
+    getMockCall('/v4/appcatalogs/', appCatalogsResponse);
+
+    nock(global.config.passageEndpoint)
+      .get('/invites/')
+      .reply(StatusCodes.Ok, invitesResponse);
+  });
+
+  // Stop persisting responses
+  afterEach(async () => {
+    await wait(() => {
+      const isDone = nock.isDone();
+
+      if (!isDone) {
+        // eslint-disable-next-line no-console
+        console.error('Nock has pending mocks:', nock.pendingMocks());
+      }
+      expect(isDone).toBeTruthy();
+    });
+
+    nock.cleanAll();
+
+    forceRemoveAll();
+  });
```

Comment:
I think this will result in unnecessary `console.error`s.
I am using this:

```javascript
afterEach(async () => {
  await wait(() => expect(nock.isDone()).toBe(true));
  nock.cleanAll();
  forceRemoveAll();
});
```